package.json

```diff
@@ -35,6 +38,7 @@
     "select2": "~3.5.2-browserify"
   },
   "scripts": {
-    "lint": "eslint -c ./.eslintrc webpack/assets/**/*.js  || exit 0"
+    "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/assets/**/*.js",
```

Comment:
$? is 0 after running eslint w/o errors . I also added the node_modules portion because otherwise it might run a globally installed one